source/draft_api.rst

```diff
@@ -172,47 +185,151 @@ GET Endpoints
       "date_worked":2014-06-12,
       "created_at":2014-06-12,
       "updated_at":2014-06-13,
+      "deleted_at": null,
       "uuid": c3706e79-1c9a-4765-8d7f-89b4544cad56,
       "revision": 3,
-      "id": 1
     }
 
-In addition, the endpoint at ``/times`` also supports several querystring parameters: user,
-project, activity, and date range. These are accessed via ``/times?user=:username``,
-``/times?project=:projectslug``, ``/times?activity=:activityslug``, ``/times?start=:date``, and
-``/times?end=:date`` (note that dates are in ISO-8601 format). When multiple different
-parameters are used, they narrow down the result set (for example,
-``/times?user=example-user&activity=dev`` will return all time entries which were entered by
-example-user AND which were spent doing development). When the same parameter is repeated,
-they expand the result set (for example, ``/times?activity=gwm&activity=pgd`` will return all
-time entries which were either for gwm OR pgd). Date ranges are inclusive on both ends.
+In addition, the endpoint at ``/times`` also supports several querystring
+parameters:
+
+* user,
+* project,
+* activity, and
+* date range.
+
+These are accessed via
+
+* ``/times?user=:username``: Filters based on username
+* ``/times?project=:projectslug``: Filters based on project slugs
+* ``/times?activity=:activityslug``: Filters based on activity slug
+* ``/times?start=:date``: Filters to dates after and including the given date.
+* ``/times?end=:date``:  Filters to dates after and including the given date.
+* ``/times/?archived=:bool``: Filters on objects with ``deleted_at`` set to
```

Comment:
The more I look at the term 'archived', the less I like it. Now that we are dealing with revisions, perhaps 'revisions' would be more apropos?

Also, this sentence says this filters on deleted_at that either is set or isn't set, which makes no sense. And the response actually has nothing to do with deleted_at being set, it just returns any extant revisions of a time.